web/assets/css/transmission-app.scss

```diff
@@ -65,7 +65,7 @@ $image-play-circle-idle: '../img/play-circle-idle.svg';
   cursor: pointer;
   display: inline-block;
   padding: 3px;
-  user-select: none;
+  -webkit-user-select: none;
```

Comment:
I’d not delete `user-select`, but do add `-webkit-user-select`.

Reply:
Alright :) I've put the non-vendor prefix version back alongside vendor prefixes they were replaced with.